boards/openlabs-kw41z-mini/board.c

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2017 Thomas Stilwell <stilwellt@openlabs.co>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     boards_openlabs-kw41z-mini
+ * @{
+ *
+ * @file
+ * @brief       Board specific initialization for openlabs-kw41z-mini
+ *
+ * @author      Thomas Stilwell <stilwellt@openlabs.co>
+ *
+ * @}
+ */
+
+#include "board.h"
+#include "periph/rtc.h"
+#include "periph/gpio.h"
+
+void board_init(void)
+{
+    /* initialize the CPU core */
+    cpu_init();
+
+    /* initialize and turn off LEDs */
+    LED0_OFF;
+    gpio_init(LED0_PIN, GPIO_OUT);
```

Comment:
It offends me too but this order is needed to avoid unintended momentary assertions. Of course in this case it hardly matters but as a rule it has to be done that way for the output to be deterministic.

Reply:
We should really add another flag to `gpio_init()` so it can be initialised to a defined state…